src/contract/compiler/Http.ts

```diff
@@ -81,6 +81,25 @@ export default class CompilerHttp extends CompilerBase {
     throw new NotImplementedError('File system access, use CompilerHttpNode instead');
   }
 
+  async generateAciBySourceCode(
+    sourceCode: string,
+    fileSystem?: Record<string, string>,
+  ): Promise<Aci> {
+    try {
+      return await this.api.generateACI({ code: sourceCode, options: { fileSystem } });
```

Comment:
return can also be without await

Reply:
it is because of try/catch
```js
async function foo() {
  try {
    return await Promise.reject(new Error('test'));
  } catch (error) {
    console.log('error handled');
  }
}

foo().catch(() => console.log('error thrown'));
```
error would be handled differently depending on `await` in this case

Reply:
ok, sounds good